### DISCUSSION TITLE
Issues with de-focusing terminal in vertical split pane

### DISCUSSION THREAD

**Discussion by gnp-x** [zed-industries/zed#9127] [Feature Requests]
I am having issues with changing focus between terminal and editor panes when the split is vertical. Normally I toggle the terminal with command + j when I need it, but in this case I wanted it vertically split for a specific project. Problem is, I can't use the command + k command + arrow key to swap back to the editor side like it works with other non-terminal windows. Instead in clears the terminal on command + k. And command + j just opens up another terminal, because I moved the terminal window to vertical split.

Don't know if I should formally put this as an issue yet, as I wanted to see if there were solutions first before possibly claiming it as something not intended to happen.

Anyone know how I can change focus with this set up? Thanks.

And on a similar note, is there a keybind to change focus between the file tree and editor window instead of toggling it again? I try not to use the mouse where I can.

**Comment by shiwenwang** [comment]
Hope that too.

In iTerm and VSCode，`cmd+opt+arrow` could switch the focus between terminal panels on MacOS.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links

**Reply by gnp-x** [reply]
This is incredibly disappointing to hear, as discord is not indexed on the internet. Everything is behind the walled-garden of discord and makes finding solutions impossible unless you have a discord account and are in the server itself.
